cswinrt/strings/additions/Windows.Storage.Streams/IBufferByteAccess.cs

```diff
@@ -1,7 +1,8 @@
 namespace Windows.Storage.Streams
 {
-    using global::System;
-
+    using global::System;
+
+    [global::WinRT.WindowsRuntimeType("Windows.Foundation.UniversalApiContract")]
```

Comment:
Is this a class issue that we need to look at our other additions and add it to them also if they are used in similar ways?  Or was this a one-off.